tests/functional/c/consider/consider_iterating_dictionary.py

```diff
@@ -1,4 +1,6 @@
-# pylint: disable=missing-docstring, expression-not-assigned, too-few-public-methods, no-member, import-error, no-self-use, line-too-long, useless-object-inheritance, unnecessary-comprehension
+# pylint: disable=missing-docstring, expression-not-assigned, too-few-public-methods
+# pylint: disable=no-member, import-error, no-self-use, line-too-long, useless-object-inheritance
+# pylint: disable=unnecessary-comprehension, use-dict-literal
```

Comment:
```suggestion
# pylint: disable=unnecessary-comprehension, use-dict-literal,invalid-name
```

Reply:
I think it would be better to change the name than add an additional disable here.
However, I don't see `var` in `bad-names` in `./pylintrc`.

Furthermore, this doesn't explain why this is only being thrown in the CI. The tests pass locally for me (and I think also for @cdce8p). Probably should try and investigate what is exactly causing this before adding a disable.
@cdce8p seems to have an idea, or at least has located the source

Reply:
I was able to replicate the failure when running under tox: (`tox -e py36 -- -k consider_iterating`). Sorry about jumping in, I was just pinged by the mention of PyCQA/astroid#979

Reply:
I forgot to add `--recreate`, which you probably did some time before. Can now indeed reproduce.

I can change this to another variable name. Can somebody tell me where `pylint` is getting the `bad-names list` from? I'm probably overlooking something..

Reply:
As to why this shows up after that merge on astroid, my reading of https://github.com/PyCQA/pylint/blob/bc95cd34071ec2e71de5bca8ff95cc9b88e23814/pylint/checkers/base.py#L1975-L1984 is that `invalid-name` is only emitted if the Assign `.value` can be inferred. Since `Compare` nodes are only just now inferred, these popped up.

Reply:
Seems I mixed up `invalid-name` with `disallowed-name` myself.

`var` is fine to us, but here it's inferred as constant which should be uppercase (like @DanielNoord mentioned).
https://github.com/PyCQA/astroid/pull/979 added additional inference for `compare` nodes which is why these can now correctly be inferred as constants. Thus `invalid-name` is emitted now.

Sorry for the ping @nelfin.